lib/smart_proxy_openscap/openscap_import_api.rb

```diff
@@ -0,0 +1,36 @@
+module Proxy::OpenSCAP
+  class ImportApi < ::Sinatra::Base
+    include ::Proxy::Log
+    helpers ::Proxy::Helpers
+    authorize_with_trusted_hosts
```

Comment:
With the other API we are certain that the client is in SSL mode.
Here, Foreman can be either SSL or not SSL. (8000 / 8443)
Setting to `authorize_with_trusted_hosts` locks it only to trusted Foreman hosts